lib/views/notes/infinite_scroll.dart

```diff
@@ -1,253 +1,277 @@
-                                          fontWeight: FontWeight.w400,
-                                        ),
-                                      ),
-                                      const SizedBox(height: 2),
-                                      // Text(
-                                      //   notes[i].price.toString() + "TL",
-                                      //   style: const TextStyle(
-                                      //     fontWeight: FontWeight.bold,
-                                      //     fontSize: 16,
-                                      //     color: Colors.black54,
-                                      //   ),
-                                      // ),
-                                      const SizedBox(height: 10),
-                                      Row(
-                                        crossAxisAlignment: CrossAxisAlignment.center,
-                                        children: [
-                                          Expanded(
-                                            child: Text(
-                                              getCityName(items[index].cityId ?? 0),
-                                              overflow: TextOverflow.ellipsis,
-                                              style: const TextStyle(
-                                                // fontWeight: FontWeight.bold,
-                                                fontSize: 12,
-                                                color: Color.fromARGB(177, 158, 158, 158),
-                                              ),
-                                            ),
-                                          ),
-                                          Text(
-                                            getFormattedDate(
-                                                items[index].updatedAt ?? items[index].createdAt),
-                                            overflow: TextOverflow.ellipsis,
-                                            style: const TextStyle(
-                                              // fontWeight: FontWeight.bold,
-                                              fontSize: 12,
-                                              color: Colors.grey,
-                                            ),
-                                          ),
-                                        ],
-                                      )
-                                    ],
-                                  ),
-                                )
-                              ],
-                            ),
-                          );
-                        }
-                      },
-                    ),
-                  );
-                } else {
-                  return const Center(child: Text('Пусто'));
-                }
-              },
+  Future<(List<CloudNote>, DocumentSnapshot?)> _fetchData({bool isRefresh = false}) async {
```

Comment:
this is not proper place for fetching data function, it should be placed inside service